mk_verificator/tests/ntp_sync/config.yaml

```diff
@@ -1,3 +1,5 @@
 ---
 skipped_nodes: ['saml-global-01.mosci.jiocloud.com']
 skipped_group: []
+sec_gauge_positive: 30
```

Comment:
I don't think we need 2 parameters for this. One should be enough. Also I would suggest renaming it to "time_deviation" or something like this.